src/TodoList.tsx

```diff
@@ -0,0 +1,106 @@
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+
+interface TodoListProps {
+  todos: Todo[],
+  removeTodo: React.Dispatch<React.SetStateAction<boolean>>,
+  editTodo: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  removeTodo,
+  editTodo,
+}) => {
+  const handleTodoRemoval = () => {
```

Comment:
handleTodoRemoval should accept a todo id and pass it to removeTodo. Example: const handleTodoRemoval = (id: string) => { removeTodo(id); };

Reply:
I haven't implemented it yet